### PR TITLE
Fixes #11263 - set newly seeded partition tables default

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: db/seeds.d/08-partition_tables.rb [hunk 1]
```diff
@@ -15,6 +15,8 @@
   ].each do |input|
     next if Ptable.find_by_name(input[:name])
     next if audit_modified? Ptable, input[:name]
+
+    input.merge!(:default => true)
     p = Ptable.create({
```
Comment (domcleal) [inline db/seeds.d/08-partition_tables.rb:18]: just add it to this existing hash merge?

**Comment by ares** [comment]
Just for the record, we [assign](https://github.com/theforeman/foreman/blob/develop/db/migrate/20150605103059_assign_ptables_to_taxonomies.rb) orgs and locs to existing ptables
this is only for newly seeded ones

File: db/seeds.d/08-partition_tables.rb (hunk 1 above)
Comment (ares) [inline db/seeds.d/08-partition_tables.rb:18]: I wanted to be consistent with provisioning templates seed file...

**Comment by dLobatog** [comment]
Merged as b35392c427899b6e5eaef48350ba0ea680eddb66, thanks @ares!

**Pull request closed without merging**